src/MCPClient/lib/clientScripts/upload-archivesspace.py

```diff
@@ -190,20 +191,21 @@ def upload_to_archivesspace(files, client, xlink_show, xlink_actuate, object_typ
                                   size=size,
                                   format_name=format_name,
                                   format_version=format_version,
-        )
+                                  inherit_notes=inherit_notes,)
```

Comment:
This is more style, but I prefer to have the closing ) on its own line so that next time we add a parameter we're only touching one line.